Fixed IncidentIDs argument to isAarray in IsIncidentPartOfCampaign (#17605)
demisto/content: Packs/Campaign/ReleaseNotes/3_2_7.md

#### Scripts
##### IsIncidentPartOfCampaign

> - Fixed an issue where the script was running on all incidents even though a campaign had already been found.

Reviewer: ```suggestion
- Fixed an issue where the script was running on each incident ID separately.
```
not sure what you meant by "even though a campaign had already been found", didn't it cause a timeout error?
